packages/@okta/i18n/src/properties/login.properties

```diff
@@ -26,6 +26,12 @@ country.label = Country
 # {0} is the name of the country and {1} is the country code
 country.option.label = {0} ({1})
 phone.extention.label = Extension
+select.default_value = Select an option
+# {0} is the current search query input
+select.filter_no_match = No results found for {0}
+sensitive_input.show = Show
+sensitive_input.hide = Hide
+loading_alt_text = Processing...
```

Comment:
@alexdahl-okta what do you think?

Reply:
Good question! Shelby and I leaned toward "Processing" because it seemed more specific, with less risk of over-promising.

It'll mostly be used while processing form submission, and a possible outcome is that the user's submission will be rejected and returned to them. So we didn't want to promise "Loading"; we're not necessarily loading the next view yet.

The indicator will also be used in a couple other views, like the terminal success redirect. In those cases "loading" might be a better fit, but feels like "processing" still works?

The pattern isn't quite the same, but some systems like [Carbon](https://carbondesignsystem.com/components/inline-loading/usage/#main-elements) encourage spinners to "notify to the user that their request is being processed" and encourage specific labels instead of "loading".

I'm open minded!

If we think "Processing" is viable, great point about the key mismatch... maybe the key could match processing? Or align with the Odyssey component, which [looks to have](https://odyssey.okta.design/?path=/docs/mui-components-circular-progress--indeterminate) a very generic indeterminate progress name?

Reply:
good point about how we're not necessarily loading the next view yet.  I think "Processing" is viable but i do agree the key should be changed to match it.